packages/instrumentation-openai/test/fixtures/chat-completion.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const { createOpenAIClient, runFnWithNockBack } = require('../testutils');
+
+async function main() {
+  const client = createOpenAIClient();
+  const messages = [
+    {
+      role: 'user',
+      content:
+        'Answer in up to 3 words: Which ocean contains the falkland islands?',
+    },
+  ];
+  const chatCompletion = await client.chat.completions.create({
+    model: process.env.TEST_MODEL_TOOLS,
+    // `max_tokens` because AzureOpenAI still does not support max_completions_tokens.
```

Comment:
maybe make "as of XXXX api version" and double-check if still the case?

Reply:
Using the latest `openai` (openai@4.76.1) and the latest OPENAI_API_VERSION=2024-10-01-preview, attempting to use `max_completion_tokens` results in:

```
      Error: Command failed: /Users/trentm/.nvm/versions/node/v18.20.4/bin/node ./fixtures/chat-completion.js
      /Users/trentm/el/elastic-otel-node5/packages/instrumentation-openai/node_modules/openai/error.js:45
                  return new BadRequestError(status, error, message, headers);
                         ^

      BadRequestError: 400 Unrecognized request argument supplied: max_completion_tokens
          at APIError.generate (/Users/trentm/el/elastic-otel-node5/packages/instrumentation-openai/node_modules/openai/error.js:45:20)
          at AzureOpenAI.makeStatusError (/Users/trentm/el/elastic-otel-node5/packages/instrumentation-openai/node_modules/openai/core.js:293:33)
          at AzureOpenAI.makeRequest (/Users/trentm/el/elastic-otel-node5/packages/instrumentation-openai/node_modules/openai/core.js:337:30)
          at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
        status: 400,
```

comment updated in commit b9d8e45